examples/platform/silabs/wifi/icd/ApplicationSleepManager.cpp

```diff
@@ -16,12 +16,23 @@
  ******************************************************************************/
 
 #include "ApplicationSleepManager.h"
+#include <lib/core/DataModelTypes.h>
 #include <lib/support/logging/CHIPLogging.h>
 
 namespace chip {
 namespace app {
 namespace Silabs {
 
+namespace {
+
+enum class SpecialCaseVendorID : uint16_t
+{
+    kAppleHome     = 4937,
```

Comment:
This is equal to VendorId::Apple. This enumeration only adds kAppleKeychain. Could that be part of VendorId too? Sounds unlikely, but that would avoid the ugly to_underlying cast.